### PR TITLE
Remove old commands information

### PR DESCRIPTION
**Pull request by sciencewhiz**

### CODE REVIEW

File: source/docs/software/commandbased/command-based-changes.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ This article provides a summary of changes from the original command-based frame
 Package Location
 ----------------
 
-The new command-based framework is located in the ``wpilibj2`` package for Java, and in the ``frc2`` namespace for C++.  The old command-based framework is still available in the original location. The new framework must be installed using the instructions: :ref:`docs/software/vscode-overview/3rd-party-libraries:WPILib Command Libraries`.
+The new command-based framework is located in the ``wpilibj2`` package for Java, and in the ``frc2`` namespace for C++. The new framework must be installed using the instructions: :ref:`docs/software/vscode-overview/3rd-party-libraries:WPILib Command Libraries`.
```
Comment (Daltz333) [inline source/docs/software/commandbased/command-based-changes.rst:9]: Are we getting rid of the "new" here?

File: source/docs/software/commandbased/what-is-command-based.rst [hunk 2]
```diff
@@ -78,4 +78,4 @@ Creating a Robot Project
 
 Creating a project is detailed in :ref:`docs/software/vscode-overview/creating-robot-program:Creating a Robot Program`. Select "Template" then your programming language then "New Command Robot" to create a basic Command-Based Robot program.
 
-When you create a New Command Robot project, the new command based vendor library is automatically imported. If you imported a 2019 project or created a different type of project, the old command library is imported, and it is necessary to import the new command based vendor library per :doc:`/docs/software/vscode-overview/3rd-party-libraries` and remove the old command library.
+When you create a  project, the new command based vendor library is automatically imported.
```
Comment (Daltz333) [inline source/docs/software/commandbased/what-is-command-based.rst:81]: Perhaps it's GitHub being weird, but is there supposed to be an extra space here?

File: source/docs/software/commandbased/command-based-changes.rst (hunk 1 above)
Comment (sciencewhiz) [reply source/docs/software/commandbased/command-based-changes.rst:9]: given that this article is a comparison of old and new, I don't think we can.

Comment (Daltz333) [reply source/docs/software/commandbased/command-based-changes.rst:9]: Ah yeah, that's fine.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**